README.md

```diff
@@ -1,53 +1,99 @@
 <h1 align="center">Marker</h1>
 <p align="center">
-    <br/>
-    <img height=200 src="res/logo.svg" alt="Marker logo"/>
+    <img height=200 src="./res/marker-logo-round.svg" alt="Marker logo" />
 </p>
 
 <br/>
 
-[![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/46214)
-[![CI Status](https://github.com/rust-marker/marker/actions/workflows/rust_bors.yml/badge.svg)](https://github.com/rust-marker/marker/actions?query=event%3Apush+workflow%3A%22Rust+(bors)%22)
+[![Crates.io](https://img.shields.io/crates/v/marker_api.svg)](https://crates.io/crates/marker_api)
+<!--
+FIXME(xFrednet): Add license shield, once crates.io also says:
+[![License: MIT OR Apache-2.0](https://img.shields.io/crates/l/marker_api.svg)](#license)
+-->
 
-**An experimental linting interface for Rust. Let's make custom lints a reality!**
+Marker is an experimental code analysis interface, with the goal to create a stable and user-friendly linting framework for the Rust programming language. Creating custom lints should be a straightforward process, and using them should be as simple as declaring a new dependency.
+
+Let's make custom lints and code analysis a reality!
 
 > **Note**
 >
-> The project is in the very early stages of development, some things are still missing and the API is not stable yet.
+> The project is in the early stages of development, some things are still missing and the API is not stable yet.
+>
+> A collection of limitations, goals and features is available below.
```

Comment:
Features, goals, and limitations (which is what you have later) is a better order.